src/Sarif/SarifUtilities.cs

```diff
@@ -168,15 +168,15 @@ public static int GetByteLength(char[] chars, Encoding encoding)
             chars = chars ?? throw new ArgumentNullException(nameof(chars));
             encoding = encoding ?? throw new ArgumentNullException(nameof(encoding));
 
-            string s = new String(chars);
+            string s = new string(chars);
             return GetByteLength(s, encoding);
         }
 
         public static int GetByteLength(string s, Encoding encoding)
         {
             s = s ?? throw new ArgumentNullException(nameof(s));
             encoding = encoding ?? throw new ArgumentNullException(nameof(encoding));
-            
+
             byte[] bytes = encoding.GetBytes(s);
```

Comment:
>b [](start = 12, length = 1)

Do you know which rule inserted the blank line above? I don't object to it.

Reply:
I'm not sure, looks like it inserted a line and deleted a line, resulting in no change from what I see, so I think we are fine.

---
In reply to: [388483612](https://github.com/microsoft/sarif-sdk/pull/1797#discussion_r388483612) [](ancestors = 388483612)